functions/src/handlers/workflow-cloud.ts

```diff
@@ -153,6 +153,27 @@ function appFactory({db, auth}: { db: any, auth: admin.auth.Auth }) {
     res.send(result);
   });
 
+  app.get('/flows/', async (req, res) => {
+    const docs = await db.collectionGroup('workflows').get();
+    const result: Array<any> = [];
+    docs.forEach(doc => {
+      let data = doc.data();
+      let unEncode = JSON.parse(data._encoded);
```

Comment:
Why use ._encoded and have to decode it ourselves?
I suspect  `data()` should be used instead of having to decode it.
I know it matches a pattern from the existing code but I suspect the existing code should not be using internal objects either (internal as marked by the `_` prefix.
I think this is a relevent example:
https://firebase.google.com/docs/firestore/query-data/get-data#get_multiple_documents_from_a_collection
I assume it can return a regular dictionary object in the same way.
Behaviour of internal objects may change, I couldn't find _encoded in a quick search of the Firebase documentation.

Reply:
The _encoded is actually the name of the field. We're storing the flow as a single JSON "encoded" string in the Firebase document.

Reply:
Comment below hopefully explains history of this

Reply:
Ahh okay! That changes my view of this then.